games-emulation/pcsx2/Manifest

```diff
@@ -1,2 +1,2 @@
 DIST pcsx2-1.4.0.tar.gz 15108326 BLAKE2B 6601019f9bd038d8cb3ed5ebd47daa5b645e537d028581c1235f9ab03f59d281639913ef037be3ef380a76bd48ea4f8728b0339fafcd331b4cfa41681dbcabd0 SHA512 87ae8bdebcaf015ba03c23056bcab32b13d6726a6c3f05e62ef54f472114399debfcfcda4208b066519c2d20345a1bb8c55582eb342dccc168f410687f3766fb
-DIST pcsx2-1.6.0_rc.tar.gz 17914747 BLAKE2B e088997a7dc8632070f6bcb70defe0ea226fe250bcefd76ab7499d765ba38e7c249fa4e7048815c173fb2e529ef9cd7e139e03b5a178e5dbcaacab6442bb5517 SHA512 41da55efe0652dc2494c7266375a00f7942f45cfbc2602ab981739102876781c474e54758a9c556f189f7b2e34c81fd1f027cfbc4033fb3b817d63387a9c9db3
+DIST pcsx2-1.6.0.tar.gz 17963332 BLAKE2B d1370fd089e1f8716a3365b9f499fef9418c4a9464e701cf0dbac8296cf59a2c6fc9b85ed680c514d8128da7b5c47aa8a067d48cac9eb0482ef2ba5f9234e60e SHA512 0a3e01a25384719fc72b636a1e9cf1cd966d0c978c1c3adfb46d6b3f76f43a25f90ef63d27a1d8092cdf444c5dc7560ce807ca81908d11ade9715e7eaaad6253
```

Comment:
Don't use `git mv`, but do a commit for adding a new ebuild, and commit for removing an old ebuild.